ocsp/ocsp.go

```diff
@@ -88,11 +88,9 @@ func Fetch(ctx context.Context, logger *log.Logger, responders []string, client
 		if resp.StatusCode != 200 && resp.StatusCode != 304 {
 			logger.Err("[fetcher] Request for '%s' got a non-200 response: %d", req.URL, resp.StatusCode)
 			backoffSeconds = 10
-			if resp.StatusCode == 503 {
-				if retryAfter := resp.Header.Get("Retry-After"); retryAfter != "" {
-					if seconds, err := strconv.Atoi(retryAfter); err == nil {
-						backoffSeconds = seconds
-					}
+			if retryAfter := resp.Header.Get("Retry-After"); retryAfter != "" {
+				if seconds, err := strconv.Atoi(retryAfter); err == nil {
+					backoffSeconds = seconds
```

Comment:
It might make sense to do a quick sanity check on the `seconds` value before assigning it to `backoffSeconds`. It needs to be a positive integer. AFAICT if `backoffSeconds` is negative the `time.NewTimer` on line 60 doesn't panic but continues in a strange state that might break in non-obvious ways.